xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/autosave.css

```diff
@@ -20,4 +20,4 @@
 #body #autosaveControl label.frequency, #body #autosaveControl label.frequency span {
   text-transform: none;
   font-weight: normal;
-}
```

Comment:
Better keep the newline at the end of the file.

Reply:
sorry, I didn't get it. Do you mean a blank new line at the end?